### PR TITLE
Blog for Byte Quantized vectors in OpenSearch

### PR DESCRIPTION
**Pull request by naveentatikonda**
### Description
Blog for Byte Quantized vectors in OpenSearch
 
### Issues Resolved
https://github.com/opensearch-project/project-website/issues/1853

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2023-10-04-byte-quantized-vectors-in-opensearch.md [hunk 1]
```diff
@@ -0,0 +1,556 @@
+---
+layout: post
+title:  "Byte-quantized vectors in OpenSearch"
+authors:
+  - navtat
+  - vamshin
+  - macrakis
+  - talw
+date: 2023-10-04 00:00:00 -0700
```
Comment (pajuric) [inline _posts/2023-10-04-byte-quantized-vectors-in-opensearch.md:9]: Please update the date to 2023-10-10

Comment (naveentatikonda) [reply _posts/2023-10-04-byte-quantized-vectors-in-opensearch.md]: Updated

File: _posts/2023-10-10-byte-quantized-vectors-in-opensearch.md [hunk 2]
```diff
@@ -0,0 +1,556 @@
+---
+layout: post
+title:  "Byte-quantized vectors in OpenSearch"
+authors:
+  - navtat
+  - vamshin
+  - macrakis
+  - talw
+date: 2023-10-10 00:00:00 -0700
```
Comment (pajuric) [inline _posts/2023-10-10-byte-quantized-vectors-in-opensearch.md:9]: Update date to 10/9/23

**Comment by pajuric** [comment]
@krisfreedain @nateynateynate @dtaivpp - Blog is approved and ready to go. The date does need to be updated to 10/9/23

**Review by nateynateynate — APPROVED** [review]
Pushed a quick change to alter the date on the filename slug as well as in the front matter. Let's get it out!

**Pull request merged**